### PR TITLE
feat: Add functional tests sd-cmd require-or build-cache

### PR DESCRIPTION
**Pull request by ibu1224**
## Context

This PR adds minimal functional testing to maintain the quality of the following features. 
1. require-or
2. sd-cmd
3. build-cache

## Objective

In this PR, add the feature file and code for each feature.

### require-or
This test checks if the OR trigger is working.

Here are the repositories that actually work.
https://github.com/screwdriver-cd-test/functional-require-or

### build-cache
Testing cache functionality at the job, event, and pipeline levels.
Build log messages are used to determine the job level cache. 
Job-level scenarios are tested using build log messages. 

Here are the repositories that actually work.
https://github.com/screwdriver-cd-test/functional-build-cache

### sd-cmd
Add new Trusted command and validation functional tests.
The Trusted Command test is based on the [Trusted Template](https://github.com/screwdriver-cd/screwdriver/blob/99e19eec4d8eab0e77cafd7998eab4b7c761a084/features/templates.feature#L82-L93).
Trusted Command must be prepared before executing the functional test.

Here are the repositories that actually work.
https://github.com/screwdriver-cd-test/functional-build-cache
## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by ibu1224** [comment]
It appears that cd.screwdriver.cd and beta are broken.

The following preparations are required for a successful functional test.
1. start a publish-trusted job in [beta](https://beta.cd.screwdriver.cd/pipelines/5457/events/93914) and [prod](https://cd.screwdriver.cd/pipelines/3894/events/715478)
2. access the command detail screen created in 1. and make it trusted
3. start the publish-distrusted job in [beta](https://beta.cd.screwdriver.cd/pipelines/5457/events/93914) and [prod](https://cd.screwdriver.cd/pipelines/3894/events/715478)

**Review by itleigns — DISMISSED** [review]

File: features/support/sdapi.js [hunk 1]
```diff
@@ -292,10 +292,36 @@ function cleanupBuilds(config) {
     });
 }
 
+/**
+ * Retrun specific build log
```
[the remark this answers is not in the document]
Comment (ibu1224) [reply features/support/sdapi.js:296]: Thank you.

**Review by tkyi — APPROVED** [review]

**Comment by ibu1224** [comment]
@tkyi 
PR build is working!

**Comment by tkyi** [comment]
Pipeline is blocked at the moment :\

**Pull request merged**